sonic_data_client/virtual_db.go

```diff
@@ -119,7 +119,7 @@ func initCountersPfcwdNameMap() error {
 }
 
 // Get the mapping between sonic interface name and oids of their PFC-WD enabled queues in COUNTERS_DB
-func getPfcwdMap() (map[string]map[string]string, error) {
+func GetPfcwdMap() (map[string]map[string]string, error) {
```

Comment:
All other function names use camelcase format, should we go back to the previous name?

Reply:
Making this function public for testing. Capitalizing the first letter in the function name makes it public.